spec/rubocop/config_spec.rb

```
@@ -174,6 +174,37 @@
       end
     end
 
+    context 'when the configuration includes multiple valid EnforcedStyle' do
```

Should we raise an error if the cop doesn't support multiple style explicitly?